src/Navigation/Dropdown/Dropdown.test.tsx

```diff
@@ -0,0 +1,330 @@
+        <DropdownItem value="Product Manager">Product Manager</DropdownItem>
+        <DropdownItem value="Software Engineer">Software Engineer</DropdownItem>
+      </Dropdown>
+    );
+
+    const optionPM = queryByText('Product Manager');
+    const optionSE = queryByText('Software Engineer');
+
+    expect(optionPM).not.toBeVisible();
+    expect(optionSE).not.toBeVisible();
+  });
+});
+
+describe('<Dropdown/> with DropdownHeader props', () => {
+  // TODO: find a way to test showHoverLine, which needs pseudo element style validation
+  test('disabled', () => {
+    render(
+      <Dropdown label="Career" disabled>
+        <DropdownItem value="Product Manager">Product Manager</DropdownItem>
+        <DropdownItem value="Software Engineer">Software Engineer</DropdownItem>
+      </Dropdown>
+    );
+    expect(document.querySelector('.dropdown-content')).toHaveStyle(
+      'cursor: not-allowed'
+    );
+  });
+
+  test('showFullWidth', () => {
+    render(
+      <Dropdown label="Career" showFullWidth>
+        <DropdownItem value="Product Manager">Product Manager</DropdownItem>
+        <DropdownItem value="Software Engineer">Software Engineer</DropdownItem>
+      </Dropdown>
+    );
+    expect(document.querySelector('.dropdown-content')).toHaveStyle(
+      'width: 100%; padding: 10px 0 10px 20px;'
+    );
+  });
+});
+
+describe('<Dropdown/> with DropdownBody props', () => {
+  test('dropDownPlacement', () => {
+    render(
+      <Dropdown label="Career" dropDownPlacement="right">
+        <DropdownItem value="Product Manager">Product Manager</DropdownItem>
+        <DropdownItem value="Software Engineer">Software Engineer</DropdownItem>
+      </Dropdown>
+    );
+    expect(queryByRole(document.body, 'listbox')).toHaveStyle(
+      'right: 0; width: auto;'
+    );
+  });
+
+  test('noLineBreak', () => {
+    render(
+      <Dropdown label="Career" noLineBreak>
+        <DropdownItem value="Product Manager">Product Manager</DropdownItem>
+        <DropdownItem value="Software Engineer">Software Engineer</DropdownItem>
+      </Dropdown>
+    );
+    expect(queryByRole(document.body, 'listbox')).toHaveStyle(
+      'white-space: normal'
+    );
+  });
+
+  test('showFullWidth', () => {
+    render(
+      <Dropdown label="Career" showFullWidth>
+        <DropdownItem value="Product Manager">Product Manager</DropdownItem>
+        <DropdownItem value="Software Engineer">Software Engineer</DropdownItem>
+      </Dropdown>
+    );
+    expect(queryByRole(document.body, 'listbox')).toHaveStyle(
+      'width: calc(100% - 20px); left: 10px;'
+    );
+  });
+
+  test('showHoverLine', () => {
+    render(
+      <Dropdown label="Career" showHoverLine>
+        <DropdownItem value="Product Manager">Product Manager</DropdownItem>
+        <DropdownItem value="Software Engineer">Software Engineer</DropdownItem>
+      </Dropdown>
+    );
+    expect(queryByRole(document.body, 'listbox')).toHaveStyle(
+      'margin-top: 13px;'
```

Comment:
hmm, test cases are spec of a module, and 'margin-top: 13px;' is the current spec. When spec changes, test needs to be updated, because they are the same thing. So I think specifying `13px` is the correct way. If spec of a module changes and we don't need to update its test file, then I think the test file is not be written correctly haha.